doc/dev_guide/internals/box_protocol.rst

```diff
@@ -265,6 +273,14 @@ be checked if it is absent.
 In a response header IPROTO_SCHEMA_VERSION is always present, and it is up to
 the client to check if it has changed.
 
+..  _box_protocol-iproto_stream_id:
+
+**IPROTO_STREAM_ID** = 0x0a.
+An unsigned number that should be unique in every stream.
+In requests IPROTO_STREAM_ID is optional and is only useful for ensuring that requests within transactions are done in separate groups.
```

Comment:
There are two purpose of streams:
1.  ensure strictly consistent execution of requests (regardless of whether they are executed within a transaction).
2.  implement interactive transactions over iproto streams (main purpose).
You mention here only second one (you right it's main purpose). But may be say about first one also?

Reply:
Okay, in my second commit, I mentioned the first, with approximately the words that you suggested.